bundle/deploy/terraform/convert.go

```diff
@@ -234,36 +234,54 @@ func TerraformToBundle(state *tfjson.State, config *config.Root) error {
 		case "databricks_job":
 			var tmp schema.ResourceJob
 			conv(resource.AttributeValues, &tmp)
+			if config.Resources.Jobs == nil {
```

Comment:
Why these were not needed before? Shall we add an unit test for this?

Reply:
Yes, we have a bug right now - for example if the remote state has a job, but the local bundle config has no jobs, then executing `bundle run` command will panic due to us assigning to a nil var. It's an edge case that can happen when you have two instances of the same project pointing to the same workspace (on two different machines for example) and then you update something in only one project (and deploy it). I'll add tests to this function

Reply:
Won't it still panic because in the next line we are potentially trying to reference a non existent job from the newly created empty job object?

Reply:
The `cur` can be nil, but the logic handles that case fine

Reply:
Added tests in the latest revision